tests/service/connectors/test_saas_connector.py

```diff
@@ -0,0 +1,72 @@
+from typing import Dict, Any, Set, Optional
+from requests import Response
+from src.fidesops.schemas.saas.saas_config import SaaSConfig
+from starlette.status import HTTP_404_NOT_FOUND, HTTP_200_OK
+import pytest
+import json
+
+from fidesops.util.data_category import DataCategory
+from fidesops.service.connectors.saas_connector import SaaSConnector
+from fidesops.schemas.saas.saas_config import SaaSRequest
+from fidesops.schemas.saas.shared_schemas import HTTPMethod
+
```

Comment:
Since you've created this new file (good idea), I'd consolidate some tests that should also belong here:

see tests/integration_tests/test_connection_configuration_integration.py 

There's a `class TestSaasConnector` and a `class TestSaaSConnectorMethods` that could probably be moved to this new file.

Reply:
oh nice, hadn't seen those. thanks for pointing that out.